### ISSUE TITLE
Introduce first test, to test PrimeFaces components

### ISSUE THREAD

**Issue by tandraschko** [primefaces-extensions/primefaces-arquillian#14]
@rmpestano @krichter722 
Are you interessted in doing this?
I would add an new project "primefaces-arquillian-tests", which uses primefaces-arquillian, to test components

I got a nice testcase today, which is pretty easy:
https://github.com/primefaces/primefaces/issues/4009
1) click on the button
2) check if modal div is below the document.body

i think it's quite easy but requires to setup the project + adding a dialog component to primefaces-arquillian

**Comment by tandraschko** [comment]
many many of the bugs are so simple cases
i think it would really boost the stability

**Comment by tandraschko** [comment]
+ i think adding new "testcases" in future just takes 1-5 minutes if the case is simple as above

**Comment by krichter722** [comment]
@tandraschko I agree that this would be really awesome. However I'm looking and applying for a new job and will have to reduce my open source project activities.
